README.md

```diff
@@ -47,6 +47,21 @@ Books:
 >  
 > The old gods have no power in the south, Stark’s family is split and there is treachery at court. Worse, the vengeance-mad heir of the deposed Dragon King has grown to maturity in exile in the Free Cities. He claims the Iron Throne.
 
+
+#### [A Court of Thorns and Roses](https://www.goodreads.com/series/104014-a-court-of-thorns-and-roses) *by [Sarah J. Maas](https://en.m.wikipedia.org/wiki/Sarah_J._Maas)* [4.5]
+
+A series of books following the adventures of 19-year-old Feyre in a world divided in two - the human side, and the High Fae side, full of magic and dangers.
+
+Books:
+ * [A Court of Thorns and Roses](https://www.goodreads.com/book/show/16096824-a-court-of-thorns-and-roses)
+ * [A Court of Mist and Fury](https://www.goodreads.com/book/show/17927395-a-court-of-mist-and-fury)
+ * [A Court of Wings and Ruin](https://www.goodreads.com/book/show/23766634-a-court-of-wings-and-ruin)
+ * [A Court of Frost and Starlight](https://www.goodreads.com/book/show/31076583-a-court-of-frost-and-starlight)
+
+> I found this to be one of the rare series that just grasps my attention to the very end. Though there are slow parts in the story, the novel compensates for these with beautiful, detailed descriptions, great character interaction, and generally fine character development. 
+>
+> I've read the series multiple times and have found myself amazed at every reiteration. [@Djolo1802](https://github.com/djolo1802)
```

Comment:
I think this review is in the wrong place. Can you put it above? This shouldn't be quoted, either.